rpm/rpm.go

```diff
@@ -104,17 +104,24 @@ func buildRPMMeta(info *nfpm.Info) (*rpmpack.RPMMetaData, error) {
 		return nil, err
 	}
 
-	hostname, err := os.Hostname()
+  hostname, err := os.Hostname()
 	if err != nil {
 		return nil, err
 	}
 
+	var release = ""
+	if info.Prerelease == "" {
+		release = defaultTo(info.Release, "1")
+	} else {
+		release = fmt.Sprintf("%s.%s", defaultTo(info.Release, "0.1"), info.Prerelease)
+  }
+  
```

Comment:
this could be:

```go
var release = defaultTo(info.Release, "1")
if info.PreRelease != "" {
  release = fmt.Sprintf("%s.%s", defaultTo(info.Release, "0.1"), info.Prerelease)
}
```